frontend/ui/widget/dictquicklookup.lua

```diff
@@ -1186,7 +1186,7 @@ end
 
 function DictQuickLookup:lookupInputWord(hint)
     self.input_dialog = InputDialog:new{
-        title = _("Input lookup word"),
+        title = _("Enter search query"),
```

Comment:
"Enter search query" is very Computer English. I'm not a fan.
```suggestion
        title = _("Enter a word or phrase to look up"),
```